app/models/case/sar/standard.rb

```diff
@@ -132,10 +136,11 @@ def initial_deadline
     end
   end
 
-  def extend_deadline!(new_deadline)
+  def extend_deadline!(new_deadline, new_extended_times)
     self.update!(
       external_deadline: new_deadline,
-      deadline_extended: true
+      deadline_extended: true,
+      extended_times: new_extended_times
```

Comment:
Just wondering what type this expects? It isn't obvious to me from the variable name

Reply:
it is integer, you can see the type from the top of "jsonb_accessor" part :)